pi-manage

```diff
@@ -1033,7 +1033,8 @@ def e_import(filename, cleanup=False):
                       conditions=event.get("conditions"),
                       ordering=event.get("ordering"),
                       options=event.get("options"),
-                      active=event.get("active"))
+                      active=event.get("active"),
+                      position=event.get("position"))
```

Comment:
If we have an old exported event definition, that does not know post and pre positions, maybe we should default to "post":
~~~~python
   position=event.get("position", "post"))
~~~~

Please consider enhancing this.

Reply:
Hi @cornelinux!

Should "Post" be the default value? If an exported configuration doesn't know about position, assuming "post" could be wrong. Anyway, if yes, I'll do it.

Reply:
If it does not know about positions, the export is from a time, when *all* event handlers where post. (No other positions have been available) 
So post would always be right in this case.

Reply:
Thanks for the explanation!

I have just pushed the change